src/components/Skill.vue

```diff
@@ -52,4 +134,71 @@ export default {
   color: paleturquoise;
   font-family: 'M PLUS Rounded 1c', sans-serif;
 }
+
+#skillCategories { padding: 20px 0; }
+
+.skillCategoriesList {
+  display: inline-block;
+  margin: 0 10px;
+}
+
+#front {
+  color: #b51a1a;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+#back {
+  color: #0f8839;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+#dev {
+  color: #571083;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+#skillList {
+  width: auto;
+  font-size: 18px;
+  border-bottom: 2px solid #f3f3f3;
+  padding: 10px 0 50px 0;
+}
+
+li {
+  display: inline-block;
+  margin: 10px 10px;
+}
+
+#front-end li {
+  color: rgba(181, 26, 26, 0.75);
+  height: 30px;
+  font-weight: bold;
+  padding: 12px 20px 2px 20px;
+  box-shadow: 0 0 8px gray;
+}
+
+#back-end li {
+  color: rgba(15, 136, 57, 0.75);
+  height: 30px;
+  font-weight: bold;
+  padding: 12px 20px 2px 20px;
+  box-shadow: 0 0 8px gray;
+}
+
+#devops li {
+  color: rgba(87, 16, 131, 0.75);
+  height: 30px;
+  font-weight: bold;
+  padding: 12px 20px 2px 20px;
+  box-shadow: 0 0 8px gray;
+}
+
+#front-change li { background-color: rgba(181, 26, 26, 0.4); }
+
+#back-change li { background-color: rgba(15, 136, 57, 0.4); }
+
+#dev-change li { background-color: rgba(87, 16, 131, 0.4); }
```

Comment:
classを指定したいはずなので、ここは
```suggestion
.dev-change li { background-color: rgba(87, 16, 131, 0.4); }
```
上の二つも同じ